crypto/rand/rand_lib.c

```diff
@@ -46,7 +46,7 @@ void rand_read_tsc(RAND_poll_fn cb, void *arg)
     unsigned char c;
     int i;
 
-    for (i = 0; i < 10; i++) {
+    for (i = 0; i < TSC_READ_COUNT; i++) {
         c = (unsigned char)(OPENSSL_rdtsc() & 0xFF);
```

Comment:
> what should be added for the TSC check, since I can't find anything in the ia32cap.pod file.

??? Quoting pod:
```
=item bit #4 denoting presence of Time-Stamp Counter.
```